index.html

```diff
@@ -191,6 +191,7 @@ <h2>Principles</h2>
 we will consider what harm it could do to society or groups,
 especially vulnerable people.
 We will prioritize potential benefits for users over potential benefits to web developers,
+user agents, advertisers or others in the ecosystem,
```

Comment:
maybe also mention "content providers" or something like it (perhaps either before or after "user agents"?)

Reply:
Should we clarify "end users" as well?

Reply:
```suggestion
We will prioritise potential benefits for end users over potential benefits to web developers, content providers, user agents, advertizers or others in the ecosystem,
```

Reply:
How's this: "We will prioritise potential benefits for end users over potential benefits to web developers, content providers, user agents, advertizers or others in the ecosystem,"
?

Reply:
Sounds good to me, modulo leaving the spelling as it was before (prioritize, advertisers), at least if we're hoping to keep the document in US English.